src/base/par_iter.rs

```diff
@@ -0,0 +1,156 @@
+//! this module implements parallelators to make matrices work with
+//! the rayon crate seamlessly
```

Comment:
Some typos and typographical things here. Maybe replace with

> Parallel iterators for matrices compatible with `rayon`.

Reply:
done.